web/src/app/pages/create-survey/task-details/task-input/task-input.component.html

```diff
@@ -13,11 +13,18 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<form [formGroup]="taskGroup" class="question-container">
-  <mat-icon cdkDragHandle class="drag-icon">drag_handle</mat-icon>
-  <div fxLayout>
+<form [formGroup]="formGroup" class="task-input">
+  <div class="task-toolbar">
+    <mat-icon>{{ taskGroup === TaskGroup.PHOTO ? 'photo_camera' : 'forum'}}</mat-icon>
```

Comment:
Optional: this won't work once we add the next button type. Why don't we add the mapping you were suggesting in this PR already?

Otherwise, leave it for the next one

Reply:
done